tests/NHSD.BuyingCatalogue.Organisations.Api.IntegrationTests/Steps/OrganisationsSteps.cs

```diff
@@ -138,6 +142,87 @@ public async Task ThenTheResponseContainsValidLocationHeaderForUser(string organ
+        }
+
+        [When(@"a GET request for UnrelatedOrganisations is made for an Organisation with name (.*)")]
+        public async Task WhenAGetRequestForUnrelatedOrganisationsIsMadeForAnOrganisationWithNameOrganisations(string organisationName)
+        {
+            var organisationId = GetOrganisationIdFromName(organisationName);
+
+            await request.GetAsync(organisationUrl, organisationId, unrelatedOrganisationsPath);
+        }
+
+        [When(@"a POST request to RelatedOrganisations is made to add an Organisation with name (.*) as a child of an Organisation with name (.*)")]
+        public async Task WhenAPostRequestForRelatedOrganisationsIsMadeToAddAnOrganisationWithNameOrganisationsAsAChildOfAnOrganisationWithNameOrganisation(string childOrganisationName, string parentOrganisationName)
+        {
+            var parentOrganisationId = GetOrganisationIdFromName(parentOrganisationName);
+
+            var childOrganisationId = GetOrganisationIdFromName(childOrganisationName);
+
+            await request.PostJsonAsync(organisationUrl, TransformRelatedOrganisationIdIntoPayload(childOrganisationId), parentOrganisationId, relatedOrganisationsPath);
+        }
+
+        [When(@"a DELETE request to RelatedOrganisations is made to delete the relationship between a parent Organisation with name (.*) and a child Organisation with name (.*)")]
+        public async Task WhenADeleteRequestToRelatedOrganisationsIsMadeToDeleteTheRelationshipBetweenAParentOrganisationWithNameOrganisationAndAChildOrganisationWithNameOrganisation(string parentOrganisationName, string childOrganisationName)
+        {
+            var parentOrganisationId = GetOrganisationIdFromName(parentOrganisationName);
+
+            var childOrganisationId = GetOrganisationIdFromName(childOrganisationName);
+
+            await request.DeleteAsync(organisationUrl, parentOrganisationId, relatedOrganisationsPath, childOrganisationId);
+        }
+
+        [Then(@"the RelatedOrganisation is returned with the following values")]
+        public async Task ThenTheRelatedOrganisationIsReturnedWithTheFollowingValues(Table table)
+        {
+            var expectedRelatedOrganisation = table.CreateSet<RelatedOrganisationTable>().FirstOrDefault();
+
+            JToken responseBody = (await response.ReadBodyAsJsonAsync())?.FirstOrDefault();
+
+            var organisation = CreateRelatedOrganisation(responseBody);
+
+            organisation.Should().BeEquivalentTo(expectedRelatedOrganisation, options => options.WithStrictOrdering());
+        }
+
+        [Then(@"a list of Related Organisations is returned with the following values")]
+        public async Task ThenAListOfRelatedOrganisationsIsReturnedWithTheFollowingValues(Table table)
+        {
+            var expectedUnrelatedOrganisations = table.CreateSet<RelatedOrganisationTable>().ToList();
+
+            JToken responseBody = await response.ReadBodyAsJsonAsync();
```

Comment:
Looks like `responseBody` isn't used. Can we remove?

Reply:
oops, left in from debugging